typings/index.d.ts

```diff
@@ -2758,22 +2732,22 @@ declare module 'discord.js' {
   }
 
   interface MessageEditOptions {
-    content?: StringResolvable;
+    content?: string;
     embed?: MessageEmbed | MessageEmbedOptions | null;
     code?: string | boolean;
     flags?: BitFieldResolvable<MessageFlagsString>;
     allowedMentions?: MessageMentionOptions;
   }
 
   interface MessageEmbedAuthor {
-    name?: string;
+    name: string;
     url?: string;
     iconURL?: string;
     proxyIconURL?: string;
   }
 
   interface MessageEmbedFooter {
-    text?: string;
+    text: string;
```

Comment:
Why are these now required? An empty object is technically valid.

Reply:
oversight, fixed in #92636cc40a59cc98f376eac520a0ddc0dbef700b

Reply:
This has been undone